client/lib/analytics/fullstory.js

```diff
@@ -42,6 +42,14 @@ export function recordFullStoryEvent( name, _props ) {
 }
 
 function maybeAddFullStoryScript() {
+	// Don't record user activity on Checkout pages
+	if ( document.location.href.indexOf( 'checkout' ) !== -1 ) {
```

Comment:
Does this ever get loaded in server side rendered paths like /themes?

If it does, checking for `document` existing would be good for safety.

Nit; you could use `.includes()` - up to you!

Reply:
Although this is probably uncommon, but this will also exclude any domain containing `checkout`.
I'd suggest checking for the `document.location.pathname` instead.